io.github.pemsley.coot.yaml

```diff
@@ -0,0 +1,410 @@
+        sha256: 7985975b787340124786f092b3a07d594b2e9cd53bbfe5f3d9b1daee7d55f56f
+
+  - name: maeparser
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/tmp/maeparser
+      - cp -r * /app/tmp/maeparser
+    sources:
+      - type: git
+        url: https://github.com/schrodinger/maeparser
+        tag: v1.3.1
+
+  - name: coordgenlibs
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/tmp/coordgen
+      - cp -r * /app/tmp/coordgen
+    sources:
+      - type: git
+        url: https://github.com/schrodinger/coordgenlibs
+        tag: v3.0.2
+
+  - name: RingDecomposerLib
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/tmp/RingDecomposerLib
+      - cp -r * /app/tmp/RingDecomposerLib
+    sources:
+      - type: git
+        url: https://github.com/rareylab/RingDecomposerLib
+        tag: v1.1.3_rdkit
+
+  - name: pubchem-align3d
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/tmp/pubchem-align3d
+      - cp -r * /app/tmp/pubchem-align3d
+    sources:
+      - type: git
+        url: https://github.com/ncbi/pubchem-align3d
+        commit: daefab3dd0c90ca56da9d3d5e375fe4d651e6be3
+
+  - name: rapidjson
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/tmp/rapidjson-1.1.0
+      - cp -r * /app/tmp/rapidjson-1.1.0
+    sources:
+      - type: git
+        url: https://github.com/Tencent/rapidjson
+        commit: 858451e5b7d1c56cf8f6d58f88cf958351837e53
+
+  - name: rdkit
+    buildsystem: simple
+    build-commands:
+      - cp -r /app/tmp/RingDecomposerLib External/RingFamilies/
+      - cp -r /app/tmp/pubchem-align3d External/pubchem_shape/
+      - cp -r /app/tmp/maeparser External/CoordGen/
+      - cp -r /app/tmp/coordgen External/CoordGen/
+      - cp -r /app/tmp/rapidjson-1.1.0 External/
+      - mkdir build
+      - cmake -S . -B build -DCMAKE_PREFIX_PATH=/app -DCMAKE_INSTALL_PREFIX=/app -DCatch2_DIR=/app -DPYTHON_EXECUTABLE=/usr/bin/python3 -DPYTHON_INCLUDE_DIR=/usr/include/python3.12 -DRDK_BUILD_CAIRO_SUPPORT=ON -DRDK_INSTALL_INTREE=OFF -DRDK_INSTALL_COMIC_FONTS=OFF -DRDK_BUILD_MAEPARSER_SUPPORT=OFF
+      - cd build && make -j5 && make install
+    sources:
+      - type: archive
+        url: https://github.com/rdkit/rdkit/archive/refs/tags/Release_2023_09_4.tar.gz
+        sha256: abacae431bbc5882b87cc8629b7ddc02757204e854aa45b6157ec2bf45d623ef
+
+  - name: coot
+    buildsystem: autotools
+    config-opts:
+      - --with-fftw-prefix=/app
+      - --with-boost=/app
+      - --with-boost-libdir=/app/lib
+      - --with-rdkit-prefix=/app
+      - --with-enhanced-ligand-tools
+      - --with-guile
+    build-options:
+      append-path:
+        - /app/lib
+        - /app/bin
+      cxxflags: "-g -O -fPIC"
+      env:
+        CPPFLAGS: "-D_USE_BOOST=1"
+        LDFLAGS: "-L/app/lib -L/app/lib64 -L/usr/lib -lpython3.12"
+        LD_LIBRARY_PATH: "/app/lib64:/app/lib:/usr/lib"
```

Comment:
there should be nothing in `/app/lib64` and the others are in the path. This shouldn't be needed.